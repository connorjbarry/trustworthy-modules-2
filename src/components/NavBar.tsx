import React from "react";
import Link from "next/link";

const NavBar = () => {
  return (
    <div className="fixed z-[100] h-20 w-full" data-testid="nav">
      <div className="flex h-full w-full items-center justify-between px-2 2xl:px-16">
        <Link href="/">Image</Link>

        <div>
          <ul className="md:flex">
            <Link href="/">
              <li className="ml-10 text-sm uppercase hover:border-b">Home</li>
            </Link>
            <Link href="/">
              <li className="ml-10 text-sm uppercase hover:border-b">
                Packages
              </li>
            </Link>
            <Link href="/">
              <li className="ml-10 text-sm uppercase hover:border-b">About</li>
            </Link>
            <Link href="/account">
              <li className="ml-10 text-sm uppercase hover:border-b">
                Account
              </li>
            </Link>
          </ul>
        </div>
      </div>
    </div>
  );
};

export default NavBar;
