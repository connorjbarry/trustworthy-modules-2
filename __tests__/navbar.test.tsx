import NavBar from "../src/components/NavBar";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("NavBar testing", () => {
  it("should render the NavBar component", () => {
    render(<NavBar />);
    expect(screen.getByTestId("nav")).toMatchSnapshot();
  });
});
