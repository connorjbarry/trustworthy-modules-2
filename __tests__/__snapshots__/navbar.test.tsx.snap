// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar testing should render the NavBar component 1`] = `
<div
  class="fixed z-[100] h-20 w-full"
  data-testid="nav"
>
  <div
    class="flex h-full w-full items-center justify-between px-2 2xl:px-16"
  >
    <a
      href="/"
    >
      Image
    </a>
    <div>
      <ul
        class="md:flex"
      >
        <a
          href="/"
        >
          <li
            class="ml-10 text-sm uppercase hover:border-b"
          >
            Home
          </li>
        </a>
        <a
          href="/"
        >
          <li
            class="ml-10 text-sm uppercase hover:border-b"
          >
            Packages
          </li>
        </a>
        <a
          href="/"
        >
          <li
            class="ml-10 text-sm uppercase hover:border-b"
          >
            About
          </li>
        </a>
        <a
          href="/account"
        >
          <li
            class="ml-10 text-sm uppercase hover:border-b"
          >
            Account
          </li>
        </a>
      </ul>
    </div>
  </div>
</div>
`;
